R/theme.R

```diff
@@ -0,0 +1,64 @@
+#' A ggplot2 theme for IHEID style plots
+#' 
+#' @importFrom ggplot2 theme
+#' @examples 
+#' \dontrun{
+#' ggplot(iris, aes(x = Sepal.Length, y = Sepal.Width)) + 
+#' geom_point() + ggtitle("Example Plot") + theme_iheid()
+#' }
+#' @export
+theme_iheid <- function(){
+  font <- "Helvetica"
+  
+  ggplot2::theme(
+    
+    #Text format:
+    #This sets the font, size, type and colour of text for the chart's title
+    plot.title = ggplot2::element_text(family=font,
+                                       size=28,
+                                       face="bold",
+                                       color=iheid_palette("IHEID")[1]),
```

Comment:
Named element index, e.g. `iheid_palette("IHEID)["red"]`, easier to read/amend?

Reply:
Changed it to `IHEIDRed`.